improver_tests/acceptance/SHA256SUMS

```diff
@@ -562,9 +562,9 @@ e0e675885f8aaa40fb3d4cb4e025bf0010110ef2bf7589bba6e3eb7c7e6c7547  ./spot-extract
 990fa6216bc485e7b95005acef4eb66a7de8ec1dd3d8b49d99770375ea6f76c6  ./spot-extract/outputs/extract_multiple_percentiles_kgo.nc
 f5b76324e319dc1c27aaca76eb5f1299820cb5317e21f3fe23be8611e6db5eda  ./spot-extract/outputs/extract_percentile_kgo.nc
 c14e6b835baa4427ad129a5dc3bfbf47b28400527a90013e1240c2f158313bdc  ./spot-extract/outputs/extract_resampled_percentiles.nc
-30767df220ce42e8030ff89b2890ee5088f78252b6bad5689309bf63afa54fd7  ./spot-extract/outputs/fixed_lapse_rate_adjusted_multiple_percentile_kgo.nc
-6339135535fd59756df6c91665b4cfdc92d8dd1527e600363d6067feac682b0c  ./spot-extract/outputs/lapse_adjusted_multiple_percentile_kgo.nc
-ec190e3249be6588f54d44061038257e86a9f3f9fee2e1690ddcc1315f9dd379  ./spot-extract/outputs/lapse_rate_adjusted_uk_temperatures.nc
+04bf544f715dd94a8e2e52b11c369f63d700de2e2fc30c2f6239962fbb5cff64  ./spot-extract/outputs/fixed_lapse_rate_adjusted_multiple_percentile_kgo.nc
```

Comment:
Please update the docstring for `cli/spot_extract.py` to describe the changed behaviour when a lapse rate or fixed lapse rate is being applied.

Reply:
I've added some information to the spot_extract CLI to try to make a user aware of the expected behaviour.